src/plays/photoshopApp/SidebarItem.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const SidebarItem = ({ name, active, handleClick }) => {
+  const className = `sidebar-item ${active ? 'active' : ''}`;
+
+  const handleClickInternal = () => {
+    handleClick();
+  };
+
+  return (
+    <button className={className} onClick={handleClickInternal}>
+      {name}
+    </button>
+  );
+};
```

Comment:
You can directly pass the `handleClick` to `onClick` and no need to create another callback.
```suggestion
 
  return (
    <button className={className} onClick={handleClick}>
      {name}
    </button>
  );
};
```

Reply:
Changed And Committed